content/00.build/40.tooling/20.hardhat/00.introduction/02.installation.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Installation
+description: How to install zksync-hardhat for your project.
+tags: ["zksync", "hardhart", "plugins", "installation"]
+---
+
+Before exploring ZKsync development with Hardhat, ensure your environment is properly set up. Follow these steps to install the necessary tools and
+meet the prerequisites for a smooth development experience.
+
+### Prerequisites
+
+**System requirements:**
+
+- Ensure your machine meets the [system requirements](%%zk_git_repo_era-compiler-solidity%%/tree/main#system-requirements) for running ZKsync and
+Hardhat. For Windows users, using Windows Subsystem for Linux (WSL 2) is highly recommended for better compatibility.
+
+**Node.js and package manager:**
+
+- Install [Node.js](https://nodejs.org) if it's not already installed. We recommend using a version supported by Hardhat.
+- Choose a package manager: [Yarn](https://classic.yarnpkg.com/en/docs/install) or [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
```

Comment:
We can remove this line. NPM comes with node so a package manager is already available.